core/api_test.go

```diff
@@ -371,3 +363,13 @@ func toTxTemplate(ctx context.Context, inp map[string]interface{}) (*txbuilder.T
 	err = json.Unmarshal(jsonInp, tpl)
 	return tpl, err
 }
+
+type alwaysLeader struct{}
+
+func (al alwaysLeader) Address(_ context.Context) (string, error) {
```

Comment:
It's okay to write 

```go
func (alwaysLeader) Address(context.Context) (string, error)
```

(leaving out the variable name if none of the parameters in that set of parentheses are used)